pkg/proxy/kafka_test.go

```diff
@@ -173,6 +179,8 @@ func (k *proxyTestSuite) TestKafkaRedirect(c *C) {
 	defer log.Logger.SetLevel(oldLevel)
 	log.Logger.SetLevel(logrus.DebugLevel)
 
+	flowdebug.Enable()
```

Comment:
Extraneous flowdebug left in the commit?

Reply:
Right, removed.